bundles/mapping/mapmodule/service/HoverHandler.js

```diff
@@ -21,7 +23,12 @@ export class HoverHandler {
         this._mapmodule.getMap().getViewport().addEventListener('mouseout', evt => {
             this.clearHover();
         }, false);
-        this.styleFactory = this._mapmodule.getGeomTypedStyles.bind(this._mapmodule);
+        this._styleFactory = this._mapmodule.getGeomTypedStyles.bind(this._mapmodule);
+        const olLayer = new olLayerVector({ source: new olSourceVector() });
+        olLayer.set(LAYER_HOVER, true, true);
+        olLayer.setZIndex(1); // TODO: how to handle layer ordering
```

Comment:
Is this required since the layer is included in the getOlLayers() array from plugins? Or is there a problem with layer ordering even with it?